types/k6/test/browser.ts

```diff
@@ -1203,105 +1203,105 @@ async function test() {
     //
     // Request
     //
-    const request = page.goto(url).then(r => r?.request());
```

Comment:
Added `!` to the returned request to make sure we're testing the right thing. Without it, we need to use `?.` everywhere, changing the return type. This is a problem in case we want to do a change where a function _actually_ returns `undefined`. Then we get a positive on the test for the wrong reason.